CHANGELOG.md

```diff
@@ -22,6 +22,7 @@
 - Updated the `MobileWorkerConnector` to use the KCP network protocol by default.
 - Changed the `mobile_launch.json` config to use the new Runtime.
 - Updated all the launch configs to use the new Runtime.
+- Changed the way builds are processed in the Unity Editor. If you don't have build support for a particular target, that specific build target will be skipped rather than the whole build process cancelled.
```

Comment:
Can we drop the first sentence and just say something like "Changed the Unity Editor building process to skip builds with missing targets"?

Reply:
I might be cheeky and change it in #705 so I don't have to rerun CI here 😛

Reply:
Unless people have code concerns